tests/test_columns.py

```diff
@@ -131,3 +131,15 @@ def test_int_columns(global_var):
     assert list(df.recommendation.keys()) == ["Enhance", "Filter", "Generalize"]
     df.intent = [0]
     assert list(df.recommendation.keys()) == ["Enhance", "Filter"]
+
+
+def test_name_column(global_var):
+    df = pd.read_csv("lux/data/car.csv")
+    new_df = df.rename(columns={"Name": "name"})
```

Comment:
for this test, can we do a quick check that the values of the "name" column have not all been converted to None values?

Reply:
Thanks! I added a few more assert statements to check for this case!